lib/failure.sh

```diff
@@ -27,6 +27,20 @@ fail_invalid_package_json() {
   fi
 }
 
+fail_dot_heroku_node() {
+  if [ -f "${1:-}/.heroku/node" ]; then
```

Comment:
👍 I was thinking that would be fine since we're just creating new things inside of it, but it would still throw an error when we try to `mkdir`